Philips.CodeAnalysis.MaintainabilityAnalyzers/Cardinality/MethodPredicates.cs

```diff
@@ -0,0 +1,38 @@
+﻿// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using LanguageExt;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Generic;
+using System.Linq;
+using static LanguageExt.Prelude;
+
+namespace Philips.CodeAnalysis.MaintainabilityAnalyzers.Cardinality
+{
+
+	public static class MethodPredicates
```

Comment:
This dogfood finding is saying, "Hey, static classes are generally a bad idea." Note that extension method classes are allowed, and I think that's essentially what you're intending to do.